beekeeper-cleanup/src/test/java/com/expediagroup/beekeeper/cleanup/handler/GenericHandlerTest.java

```diff
@@ -0,0 +1,61 @@
+package com.expediagroup.beekeeper.cleanup.handler;
+
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import com.expediagroup.beekeeper.cleanup.path.aws.S3PathCleaner;
+import com.expediagroup.beekeeper.core.model.EntityHousekeepingPath;
+import com.expediagroup.beekeeper.core.model.PathStatus;
+import com.expediagroup.beekeeper.core.repository.HousekeepingPathRepository;
+
+@ExtendWith(MockitoExtension.class)
+public class GenericHandlerTest {
+
+  @Mock private HousekeepingPathRepository housekeepingPathRepository;
+  @Mock private S3PathCleaner pathCleaner;
+  @InjectMocks private final UnreferencedHandler handler = new UnreferencedHandler(pathCleaner);
+  @Mock private EntityHousekeepingPath mockPath;
+
+  @Before
+  public void init() {
+    handler.setS3PathCleaner(pathCleaner);
+    MockitoAnnotations.initMocks(this);
```

Comment:
Do you need to do this if you're using the `ExtendWith` annotation?

Reply:
Apparently not. At some point in working on this, injectMocks didn't seem to be populating correctly without this `initMocks` call, but that seems like it was probably due to some other error that I was attributing this to 😅. Removed.